pkg/api/k8s_versions.go

```diff
@@ -116,25 +116,25 @@ var kubernetesImageBaseVersionedImages = map[string]map[string]map[string]string
 	common.KubernetesImageBaseTypeGCR: {
 		"1.19": {
 			common.AddonResizerComponentName:  "addon-resizer:1.8.7",
-			common.MetricsServerAddonName:     "metrics-server-amd64:v0.3.6",
+			common.MetricsServerAddonName:     "metrics-server/metrics-server:v0.3.7",
 			common.AddonManagerComponentName:  "kube-addon-manager-amd64:v9.1.1",
 			common.ClusterAutoscalerAddonName: "cluster-autoscaler:v1.18.0",
 		},
 		"1.18": {
 			common.AddonResizerComponentName:  "addon-resizer:1.8.7",
-			common.MetricsServerAddonName:     "metrics-server-amd64:v0.3.6",
+			common.MetricsServerAddonName:     "metrics-server/metrics-server:v0.3.7",
 			common.AddonManagerComponentName:  "kube-addon-manager-amd64:v9.1.1",
 			common.ClusterAutoscalerAddonName: "cluster-autoscaler:v1.18.0",
 		},
 		"1.17": {
 			common.AddonResizerComponentName:  "addon-resizer:1.8.7",
-			common.MetricsServerAddonName:     "metrics-server-amd64:v0.3.6",
+			common.MetricsServerAddonName:     "metrics-server/metrics-server:v0.3.7",
 			common.AddonManagerComponentName:  "kube-addon-manager-amd64:v9.1.1",
 			common.ClusterAutoscalerAddonName: "cluster-autoscaler:v1.17.1",
 		},
 		"1.16": {
 			common.AddonResizerComponentName:  "addon-resizer:1.8.7",
-			common.MetricsServerAddonName:     "metrics-server-amd64:v0.3.4",
+			common.MetricsServerAddonName:     "metrics-server/metrics-server:v0.2.1",
```

Comment:
Why are we regressing the 1.16 metrics-server to v0.2.1 instead of leaving it at v0.3.4 for the GCR case? To narrow the number of versions supported by the VHD?

Reply:
becuz typoz